cmd/install/assets/recordingrules/hypershift.yaml

```diff
@@ -83,4 +83,10 @@ groups:
     expr: max by(platform) (hypershift_hostedclusters)
     
   - record: platform:hypershift_nodepools:max
-    expr: max by(platform) (hypershift_nodepools)
\ No newline at end of file
+    expr: max by(platform) (hypershift_nodepools)
+
+  - record: hypershift:nodepools:size
+    expr: max by(name, platform) (hypershift_nodepools_size)
```

Comment:
"name_platform:hypershift_nodepools_size:max" would be a more descriptive name. What's the expected cardinality for the name and platform labels?

Reply:
name and platform labels identify a unique nodepool, I can't tell what the expected cardinality would be. It boils down to how many nodepools are created in total in a management cluster.
assuming an average of 3 Nodepools per HostedCluster, and the maximum number of 80 HostedClusters we end up with 240 nodepools/timeseries.

Reply:
240 series is a lot and needs an explicit approval (see https://rhobs-handbook.netlify.app/products/openshiftmonitoring/telemetry.md/#request-approval).

Reply:
cc @jan--f

Reply:
And I assume the same is true for `name_platform:hypershift_nodepools_available_replicas:max`? That is  quite a lot of additional time series. 
Unless you really need this level of detail per node pool, I'd recommend further aggregation.

Reply:
changed to aggregate per HostedCluster, the maximum cardinality would between 80-100

Reply:
This is still a lot. Can't you do min/max/median + a few percentiles?

Reply:
I am not sure if min/max/median + a few percentiles would give us any real insights anymore.
cc @zanetworker

Reply:
@simonpasquier that won't provide meaningful data if we are trying to understand how one customer is using the feature. We can always min/max/... outside of the metric.